iam_openapi_spec.yml

```diff
@@ -1805,9 +1805,11 @@ components:
         description: Brief description about ACME
         rootUser:
           username: username
+          name: Lorem Ipsum
```

Comment:
Should be marked as required.

Reply:
It's required already.
https://github.com/hwslabs/iam/blob/f47370cd368a5603cc62ca646decc4382ea24f48/iam_openapi_spec.yml#L2194-L2208